src/components/AccountDropdown/AccountDropdown.tsx

```diff
@@ -315,6 +315,7 @@ export const AccountDropdown: FC<AccountDropdownProps> = memo(
     if (!isValidAccountNumber(accountNumber)) return null
     if (!Object.keys(accountIdsByNumberAndType).length) return null
     if (!accountLabel) return null
+    if (!selectedAccountId) return null
```

Comment:
q: Do we need this as part of this PR? `<AccountDropdown />` is inherently bug-prone and unless we explicitly need this, I'd revert it

Reply:
Nop we don't, removed in [d831295](https://github.com/shapeshift/web/pull/7663/commits/d83129520fef459ee13122165049b64ef0bc470d)